tracer/test/test-applications/integrations/Samples.OpenTelemetrySdk/Program.cs

```diff
@@ -145,13 +145,14 @@ private static void RunAddEventOverloads(TelemetrySpan span)
         {
             { "foo", 1 },
             { "bar", "Hello, World!" },
-            { "baz", new int[] { 1, 2, 3 } }
+            { "baz", new int[] { 1, 2, 3 } },
+            { "strings", new string[] { "str", "1" } }
```

Comment:
Can wee add a test case with invalid attribute values (ex: nested dict)?

Reply:
Good idea, added some invalid attributes in 979e4489229be731dde0e5380dec1ca71340ecda and you'll see that the snapshots have not changed